### DISCUSSION TITLE
How to update data in store

### DISCUSSION THREAD

**Discussion by snax4a** [infinitered/ignite#1850] [Q&A]
Hi,
I am trying the default boilerplate with characters api and store.

However I can't figure out how can I get a fresh data from the API on each refresh, or how can I force to update the state.
In reactotron is see that every time i refresh the app a new api call is made and the returned data is different but in the app screen im all the time getting old data from the first api call.

I know it is saved to asyncstorage but i have no idea how can disable the storage or force to update it on each api call

**Comment by jamonholmgren** [comment]
@snax4a Apologies -- just seeing this now.

If you had a repo that I could look at, I'd be more than happy to do a PR that shows you how to make that happen. As it is, I don't think I've run into this myself.
